src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPageNew.tsx

```diff
@@ -59,6 +79,40 @@ const FindStatisticsPageNew: NextPage<Props> = ({
     });
   };
 
+  const handleSearchPublications = async (nextSearchTerm: string) => {
+    await updateQueryParams({
+      ...omit(router.query, 'page'),
+      search: nextSearchTerm,
+      sortBy: 'relevance',
+    });
+
+    logEvent({
+      category: 'Find statistics and data',
+      action: 'Publications searched',
```

Comment:
Just wondering if it might be useful to include what publication search term the user has specified?

Reply:
It sends the search term as the event label.

Reply:
At right my bad, didn't notice at first glance